src/components/Blog-Footer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function BlogFooter(props) {
+  return (
+    <div className='container-fluid'>
+      <div className='row'>
+
+      </div>
+      <div className='row'>
+        <div className='col-12'>
+          Share To Social Media Buttons
+        </div>
+      </div>
+      <div className='row'>
+        <div className='col-4'>
+          Related Post #1
+        </div>
+        <div className='col-4'>
+          Related Post #2
+        </div>
+        <div className='col-4'>
+          Related Post #3
+        </div>
+      </div>
+      <div className='row'>
+         {props.post.frontmatter.tags.map((tag) => ( // ONE DAY THESE WILL BECOME LINKS TO TAG PAGES
+            <div className='col-1' key={tag}>
+              #{tag}
+            </div>
+         ))}
+      </div>
+      <div className='row'>
+        <div className='col-12'>
+          One Day, I will add a coment section...
+        </div>
+      </div>
+    </div>
+  );
+}
+
+BlogFooter.propTypes = {
```

Comment:
A few thoughts about these proptypes:

- You're not using `id` yet, or most of these other fields — I typically wouldn't include them unless they're actually referenced in the component (cc @solomonhawk for thoughts, though)
- This is probably down to personal preference, but I like to use `import { array, shape, string } from 'prop-types` so I can write the proptypes themselves as `id: string, post: shape({ ... }),` etc. Cuts down on needing to write "PropTypes a ton of times"
- I noticed that `frontmatter` and some other items in here are duplicated between files. This is pretty common; sometimes several components use the exact same type of data. In that case, it's sometimes useful to set up some shared proptypes in a file and import them:

```js
// /src/components/prop-types.js
import { array, string, shape } from 'prop-types'

export const postPropType = shape({ 
  ... 
})

// /src/components/Blog-Footer
import { postPropType } from '../prop-types'

...

BlogFooter.propTypes = {
  id: string,
  post: postPropType,
}
```

Reply:
#3 is a great idea. Some of the other fields are queried because I plan to use them in the future, but I'm sure some of them are trivial.